ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/editor/gutter/Gutters.java

```diff
@@ -17,6 +17,9 @@ private Gutters() {}
   /** Logical identifer for the breakpoints gutter. */
   public static final String BREAKPOINTS_GUTTER = "breakpoints";
 
+  /** Logical identifer for the vcs change markers gutter. */
+  public static final String VCS_CHANGE_MARKERS_GUTTER = "vcsChangeMarkersGutter";
```

Comment:
No, I don't see any problems in such name, `VCS`  that change markers are related to VCS.